packages/shared/src/components/modals/ShareModal.spec.tsx

```diff
@@ -100,8 +101,14 @@ describe('ShareModal Test Suite:', () => {
 
     expect(btn).toBeInTheDocument();
     btn.click();
+    const useRouterMock = useRouter as jest.Mock;
```

Comment:
Maybe we should refactor the names of these tests as well, as they still mention "modal"
It could become confusing down the line.